src/app/features/swap-form/selectors.ts

```diff
@@ -43,6 +46,11 @@ export const selectSwapConfirmationFlowStep = (state: RootState) =>
   state.swapForm.swapConfirmationFlow.step;
 export const selectSwapConfirmationFlowError = (state: RootState) =>
   state.swapForm.swapConfirmationFlow.error;
+export const selectSwapConfirmationFlowEtherscanLink = (state: RootState) => {
+  return state.swapForm.swapConfirmationFlow.txHash
+    ? getViewOnEtherScanLink(state.network.chainId, state.swapForm.swapConfirmationFlow.txHash)
+    : 'https://voltz.xyz';
```

Comment:
Make it etherscan.com as default

Reply:
[e2c9e01](https://github.com/Voltz-Protocol/voltz-ui/pull/580/commits/e2c9e0182796ac11827202bfb2c1334f452239d9) This redirects to https://etherscan.io/error, which I think it's fine since it should never happen